api/scheme.go

```diff
@@ -48,4 +50,6 @@ func init() {
 	apiextensionsv1.AddToScheme(Scheme)
 	kasv1beta1.AddToScheme(Scheme)
 	prometheusoperatorv1.AddToScheme(Scheme)
+	operatorsv1.AddToScheme(Scheme)
+	operatorsv1alpha1.AddToScheme(Scheme)
```

Comment:
Wanted to call out that these schemas are only required for e2e tests. There is no reason they are included in the hypershift payload. Open to removing them if people believe creating an e2e client is worthwhile.

Reply:
The e2e tests should already be obtaining clients exclusively through [GetClientOrDie()](https://github.com/openshift/hypershift/blob/main/test/e2e/util/client.go#L23), which itself has control over which `Scheme` is passed to the underlying client builder.

What if we added a new `Scheme` to the `util` package (in [client.go](https://github.com/openshift/hypershift/blob/main/test/e2e/util/client.go)) which extends [hyperv1.Scheme](https://github.com/openshift/hypershift/blob/main/api/scheme.go) to add the e2e-specific APIs?

Reply:
Seems reasonable to me.

Reply:
Done in the latest version of the API. I did not export the scheme variable as it should only be used in e2e tests.